### ISSUE TITLE
1.20.6 Fails to Initialize

### ISSUE THREAD

**Issue by GentlemanCheesy** [Anarchick/skript-packet#33]
Trying to get this Skript Addon working on my 1.20.6 testing server, however it will not run. It gives a error in the logs, ill provide it below. No rush on this, obviously 1.20.6 is pretty new! Its likely an issue with how much was changed in these two game updates. Thank you!

https://pastebin.com/AvbMg94X

**Comment by Anarchick** [comment]
I have already check the support of 1.20.6 and I don't plan to update until Paper and ProtocolLib release more stable versions. 

Sorry but we have to wait.

**Comment by GentlemanCheesy** [comment]
Any potential for a near future 1.20.6 update now that paper has been releasing more builds and ProtocolLib is more functional? No rush on this, same as before, just wanted to check in! :)

**Comment by AKtomik** [comment]
Hey !
> I have already check the support of 1.20.6 and I don't plan to update until Paper and ProtocolLib release more stable versions.
same for 1.21 ; paper is now stable, and ProtocolLib begin to be more functional.
Any release for 1.20.6/1.21 would be awsome, this addon is really useful, it would be hard to do without it ^^'
But no pressure, of course! at your own pace.

**Comment by Anarchick** [comment]
I have start working on mc 1.21 + protocolib-723 and I'm able to start the server but I have an important issue to solve before release a new version of SP

**Comment by Anarchick** [comment]
Note : Minecraft 1.21 will break a lot of stuff related to nms (and wrap from nms)

**Comment by Anarchick** [comment]
[Skript-Packet-2.3.0-SNAPSHOT.zip](https://github.com/user-attachments/files/16082831/Skript-Packet-2.3.0-SNAPSHOT.zip)
Can you try this version and tell me if you find any bugs. I have only tested for Paper 1.21#40, ProtocolLib#723 and Skript-2.9.0-pre1

I suppose this build works with Java 17+ (or maybe 21+ if not)

**Comment by GentlemanCheesy** [comment]
Thank you very much for your efforts! 
I've just finished testing this snapshot with my scripts. Regarding any bugs, so far ive only noticed one that happens when I try to get a location from the following:

![image](https://github.com/Anarchick/skript-packet/assets/68432025/b9e021ef-777d-4d3a-96c9-d20d4b0f0c51)

Currently this works almost perfect except when used outside of the overworld, it does not accurately get the world of the block the player is mining. It will show 'world' even when that's not the correct world name. See screenshot below for example of output while in The Nether.

![image](https://github.com/Anarchick/skript-packet/assets/68432025/632ab991-444c-43df-ad1a-d1688ff63271)

Nothing else to report aside from what versions I tested on:
Paper 1.20.6 Build 148
Skript-2.9.0-pre1
ProtocolLib#723
